modules/inbucket/inbucket.go

```diff
@@ -0,0 +1,73 @@
+package inbucket
+
+import (
+	"context"
+	"fmt"
+	"net"
+
+	"github.com/testcontainers/testcontainers-go"
+	"github.com/testcontainers/testcontainers-go/wait"
+)
+
+// InbucketContainer represents the Inbucket container type used in the module
+type InbucketContainer struct {
+	testcontainers.Container
+}
+
+// Smtp returns the connection string for the smtp server, using the default
+// 2500 port, and obtaining the host and exposed port from the container.
+func (c *InbucketContainer) Smtp(ctx context.Context) (string, error) {
+	containerPort, err := c.MappedPort(ctx, "2500/tcp")
+	if err != nil {
+		return "", err
+	}
+
+	host, err := c.Host(ctx)
+	if err != nil {
+		return "", err
+	}
+
+	return net.JoinHostPort(host, containerPort.Port()), nil
+}
+
+// WebInterface returns the connection string for the web interface server,
+// using the default 9000 port, and obtaining the host and exposed port from
+// the container.
+func (c *InbucketContainer) WebInterface(ctx context.Context) (string, error) {
```

Comment:
In an automated test suite, what would be the use case for this URL? is this something you plan to use in dev-environment, where all the backend services are running?

Reply:
Ok, I've read the tests now, and this URL is exposing REST APIs. 👌